src/core/settings.c

```diff
@@ -1282,6 +1310,7 @@ QuicSettingsSettingsToInternal(
     SETTING_COPY_TO_INTERNAL(MigrationEnabled, Settings, InternalSettings);
     SETTING_COPY_TO_INTERNAL(DatagramReceiveEnabled, Settings, InternalSettings);
     SETTING_COPY_TO_INTERNAL(ServerResumptionLevel, Settings, InternalSettings);
+    SETTING_COPY_TO_INTERNAL(IdleSrcCidChangeMs, Settings, InternalSettings);
```

Comment:
Please see the comment right below this. This needs to be fixed.

Reply:
I think you meant to use size check version of this macro instead of SETTING_COPY_TO_INTERNAL one